routes/workoutlog.js

```diff
@@ -1,3 +1,22 @@
+<<<<<<< HEAD
+// workoutlog data... connects to "user" page
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
const express = require("express"); 
```